src/membraneWebRTC.ts

```diff
@@ -1262,14 +1256,42 @@ export class MembraneWebRTC {
     }
   }
 
-  private getTrackIdToMetadata = () => {
-    const trackIdToMetadata = {} as any;
+  private getTrackIdToTrackInfo = () => {
+    const trackIdToMid = this.getTrackIdToMid();
+
+    const tracksInfo = {} as any;
     Array.from(this.localPeer.trackIdToMetadata.entries()).forEach(
       ([trackId, metadata]) => {
-        trackIdToMetadata[trackId] = metadata;
+        const maxBandwidth =
+          this.localTrackIdToTrack.get(trackId)!.maxBandwidth;
+        tracksInfo[trackId] = {
+          trackMetadata: metadata,
+          maxBandwidth:
+            maxBandwidth instanceof Map
+              ? Object.fromEntries(maxBandwidth)
+              : maxBandwidth,
+          mid: trackIdToMid ? trackIdToMid.get(trackId!) : null,
```

Comment:
I don't think we should allow for sending `null` here. Server won't be able to handle it so we should assert that trackIdtoMid is always present and contains `trackId`.